<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMDebit</class>
 <widget class="QMainWindow" name="IHMDebit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>561</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>561</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>561</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Circuit d'eau - Moniteur</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Cuve_Rigide">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>311</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Cuve Rigide</string>
    </property>
    <widget class="QProgressBar" name="Cuve_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>291</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar::chunk{background-color:#0000FF}</string>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="VCuve_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume : 2500 / 3535 L</string>
     </property>
    </widget>
    <widget class="QLabel" name="VCuve_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume : 2500 / 3535 L</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="Cuve_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>291</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar::chunk{background-color:#0000FF}</string>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Cuve_Souple">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>Cuve Souple</string>
    </property>
    <widget class="QProgressBar" name="Cuve_S">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>291</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar::chunk{background-color:#0000FF}</string>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="VCuve_S">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume : 5000 / 10000 L</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Ext">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Extérieure</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Sens">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Debit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Transfert instantanée:
0 L/min</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHist">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/image/note.png</normaloff>:/image/note.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Consomation">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Conso. instantanée:
0 L/s</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/image/engrenage.png</normaloff>:/image/engrenage.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>141</width>
      <height>63</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/image/logo.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="connexion">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Attention!
Arduino potentiellement déconnecté</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
